cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,51 @@
-describe('', () => {
+describe('Article flow', () => {
+  let user;
+  let article;
+
   before(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
+    cy.task('generateArticle').then((generatearticle) => {
+      article = generatearticle;
+    });
   });
 
-  it('', () => {
-
+  it('create new article', () => {
```

Comment:
should allow to create new article
check and fix test names everywhere - add "should"